### DISCUSSION TITLE
Can some one explain this to me?

### DISCUSSION THREAD

**Discussion by ToadOnbekend** [BartoszCichecki/LenovoLegionToolkit#977] [Q&A]
![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/85346493/d91402a7-16ce-483e-9c53-a723ffe9e94a)

What does the Maximum power Limit offset mean and do?

**Comment by BartoszCichecki** [comment]
As written in the description it allows to set an offset for max values for power limits. If your CPU power limit is max 80, and you set 10 in this option, you will be able to set CPU power limit to max 90.

As written, not all devices will allow/respect values like this so it's up to you to test and figure it out.
